wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -8,16 +8,15 @@
     - `search_url` - if present, display a search box. This is a URL route name (taking no parameters) to be used as the action for that search box
     - `query_parameters` - a query string (without the '?') to be placed after the search URL
     - `icon` - name of an icon to place against the title
-    - `tabbed` - if true, add the classname 'tab-merged'
```

Comment:
I still see two header template tags in the codebase that set `header=True`, and a `{% if tabbed %}` in the settings’ edit template. Can we remove all of this?